pkg/scheduler/queue/tenant_queues.go

```diff
@@ -109,7 +109,7 @@ func newTenantQueues(maxUserQueueSize int, forgetDelay time.Duration) *tenantQue
 	}
 }
 
-func (q *tenantQueues) hasTenantQueues() bool {
+func (q *tenantQueues) hasNoTenantQueues() bool {
```

Comment:
@chaudum I was a little confused by the name. I think it returns `true` if there are no tenant queues.

Reply:
good catch